src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java

```diff
@@ -62,6 +60,21 @@ public FluidConvertingInventoryAdaptor(@Nullable InventoryAdaptor invItems, @Nul
     @Nullable
     private final IInterfaceHost selfInterface;
     private final boolean onmi;
+    private static Method eioTypeCheck;
+    private static Class<?> conduitClazz;
+
+    static {
+        if (ModAndClassUtil.EIO) {
+            try {
+                conduitClazz = Class.forName("crazypants.enderio.conduit.TileConduitBundle");
+                eioTypeCheck = conduitClazz.getDeclaredMethod("getConduit", Class.class);
+            } catch (ClassNotFoundException | NoSuchMethodException e) {
+                eioTypeCheck = null;
+            }
+        } else {
+            eioTypeCheck = null;
+        }
+    }
```

Comment:
Wouldn't it be better to implement/make public necessary helper methods from EIO side than use reflection?

Reply:
compiler will ask for mek api if i use `TileConduitBundle` directly because `IConduitBundle` implement `IGasHandler` interface.